src/prefect/tasks/gsheets/gsheets.py

```diff
@@ -0,0 +1,148 @@
+from prefect.utilities.tasks import defaults_from_attrs
+import gspread
+from typing import List, Union, Any, TypedDict, Callable
+from prefect import task, Task
+import pathlib
+
+
+class GsheetUpdates(TypedDict):
+    """A Typed Dict to describe one of the dictionary elements returned after writing Rows"""
+
+    spreadsheetId: str
+    updatedRange: str
+    updatedRows: int
+    updatedColumns: int
+    updatedCells: int
+
+
+class GsheetResponse(TypedDict):
+    """A Typed Dict to describe what's returned after writing Rows"""
+
+    spreadsheetId: str
+    tableRange: str
+    updates: GsheetUpdates
+
+
+class WriteGsheetRow(Task):
+    """
+    A task for writing a row to a Google Sheet.
+    Note that _all_ initialization settings can be provided / overwritten at runtime.
+    Args:

```

Comment:
Could you add a blank line before every `Args:` and `Returns:` in the docstrings (here and all the ones below)? The docstring format should be:

```
Description

Args:
    - ...

Returns:
    - ...
```

Reply:
Added!